src/renderer/domains/collectives/referendum/resource.ts

```diff
@@ -49,6 +49,18 @@ async function parseProposal(proposal: FrameSupportPreimagesBounded, api: ApiPro
       }
     }
 
+    if (struct.method === 'send') {
```

Comment:
Not any send can be considered "whitelist". To parse this precisely we need to do the following:

1. Check that the method is `polkadotXcm.send`
2. Take the `dest` argument, convert `multiLocation` to `chainId` (we can discuss how do that right separately)
3. Check that the `message` argument, which is an xcm message, contains the `Transact` instruction. Decode the `call` field of the `Transact` instruction using `ApiPromise` correponding to the `chainId` obtained at step (2)
4. Check that the decoded `call` is `whiteList.whitelistCall`